src/Nethermind/Nethermind.Consensus.AuRa/Json/AbiTypeConverter.cs

```diff
@@ -18,7 +18,7 @@
 using Nethermind.Abi;
 using Newtonsoft.Json;
 
-namespace Nethermind.Serialization.Json.Abi
+namespace Nethermind.Consensus.AuRa.Json
```

Comment:
Why move to Aura? This is generic.

Reply:
when you move the contract code out it will be naturally together